### ISSUE TITLE
Claim LCC from TREZOR

### ISSUE THREAD

**Issue by ox1** [litecoincash-project/litecoincash#10]
Hello, when I will be able to claim my liteconcash from Trezor without exposing my private keys/recovery seeds?
 
Will you support claiming LCC from trezor wallet?  It should not be hard for you, If yes what are your ETA? A lot of people uses trezor and want to claim their LCC Asap.

Thank you in advance

**Comment by DaveKoz** [comment]
The only way for you to get your coins is if Trezor supports the fork.
It's to late now for you to transfer from Trezor to a litecoin wallet.

**Comment by ox1** [comment]
Thank you for your response.

I am sorry to disappoint you sir, but Trezor devs said that they won't support this or any future fork, BUT they said that given that ALL Trezor code is **open source** (https://github.com/trezor), it would not be hard for a **whole team of a cryptocurrency's developers** to develop a claiming tool like the one made by Bitcore (BTX) developers.

**So, Will LCC developers ever implement a claiming tool for Trezor?** Because a lot of people had their ltc on trezor at the time of the snapshot and a lot of them will eventually expose their recovery seed/ private keys to untrusted parties in order to get their LCC.

Also, The official lcc website tool to see how much lcc are linked to a wallet address IS NOT WORKING for address starting with a '3' or an 'M', saying that they are "Not a Litecoin address!". Are you serious about that?? I copy and paste my addresses directly from Trezor, **they ARE LITECOIN ADDRESSES**.

Thank you in advance.

**Comment by DaveKoz** [comment]
Yes I know Trezor will not support thats why you needed to take action before the fork.

As for address you need to convert the M address with the link that is on Trezor's site.

I'm just a guy I'm not a dev for the team I have no interest in developing a tool to unlock Trezor's coins.

**Comment by ox1** [comment]
the converter converts 'M' to '3' and '3' to 'M', I've seen no way to get an 'L' address from there. If you are not a developer then please don't reply for them, I clearly asked to devs.

Also, in general, one doesn't move hundreds of coins out of a secure hardware wallet just to get a free shitfork airdrop that doesn't even provide a splitting tool. btw there must be a reason if the price is already going to 0 despite most coins being blocked.

**Comment by DaveKoz** [comment]
You clearly have rage issues. Good luck with that.

**Comment by ox1** [comment]
yep, ty. I'll wait for a real response from real devs.

**Comment by mtrabelsi** [comment]
oh fuck cant believe that actually some people falling into the scam!

**Comment by Chadward23** [comment]
Lol what a dumbass
